background.html

```diff
@@ -6,10 +6,14 @@
 
     <!-- include background scripts -->
     <script type="text/javascript" src="privly-applications/shared/javascripts/local_storage.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/storage.js"></script>
```

Comment:
Click the unfold button in the github interface [on this page.](https://github.com/privly/privly-chrome/pull/120/files#r34321039)